mlflow/entities/span.py

```diff
@@ -300,6 +300,10 @@ class can be reused without any modification.
         self._attributes.set(SpanAttributeKey.REQUEST_ID, request_id)
         self._attributes.set(SpanAttributeKey.SPAN_TYPE, span_type)
 
+    def set_span_type(self, span_type: str):
```

Comment:
Is this applicable to / useful for any other integrations?

Reply:
Not now but I think it is not too uncommon scenario to update span type after its creation.